realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -1586,6 +1588,18 @@ public void run() {
         return new RealmAsyncTaskImpl(pendingTransaction, asyncTaskExecutor);
     }
 
+    /**
+     * Invokes transaction on worker thread and returns Completable
+     *
+     * @param transaction {@link io.realm.Realm.Transaction} to execute.
+     * @return {@link Completable} representing asynchronous transaction
+     * @throws UnsupportedOperationException if the required RxJava framework is not on the classpath.
+     */
+    public Completable executeTransactionCompletable(final Transaction transaction) {
```

Comment:
I'm not 100% sure about this name. From the discussion in https://github.com/realm/realm-java/issues/2126#issuecomment-196844283 it seems that this should be renamed to `asCompletable(final Transaction transaction). This also seems to be more in line with the naming from e.g `RealmResults.asFlowable()`.

Or did you have some other reasoning for naming it this?

@nhachicha do you have any thoughts on this?

If we indeed end up wrapping an async transaction `asCompleteteAsync` would propably fit our current naming scheme better as well...Naming is hard :smile:

Reply:
`asCompletable` feels more aligned with current API